### ISSUE TITLE
NaN loss occured with exp and sqrt learning_rate_decay_scheme

### ISSUE THREAD

**Issue by zh794390558** [tensorflow/tensor2tensor#614]
NaN loss occured with exp and sqrt learning_rate_decay_scheme, and almost come up at `warmup step`.
